lib/Conversion/ExportVerilog/ExportVerilogInternals.h

```diff
@@ -102,7 +102,7 @@ struct NameCollisionResolver {
   llvm::StringSet<> usedNames;
 
   /// Numeric suffix used as uniquification agent when resolving conflicts.
-  size_t nextGeneratedNameID = 0;
+  llvm::StringMap<size_t> nextGeneratedNameID;
```

Comment:
Using a map of Attribute to size_t may be cheaper if we are know to always use String Attr.

Reply:
It looks replacing Attribute with size_t here would require more broader refactoring so i'd like to keep using strings as key within this PR.